api/api.proto

```diff
@@ -890,6 +890,26 @@ message CapacityUsageInfo {
   int64 total_bytes = 3;
 }
 
+message VolumeUsage {
```

Comment:
There should be a comment for VolumeUsage message so that it shows up in the documentation on https://libopenstorage.github.io/w/index.html

But do not add the name of the message, just describe the message.

Example: https://libopenstorage.github.io/w/release-6.4.generated-api.html#sdkvolumecreaterequest

Reply:
Done